src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseSearchEndpoint.java

```diff
@@ -67,45 +59,25 @@ public CaseSearchResult searchCases(
             + "the search results will contain only metadata by default (no case field data). To get case data in the "
             + "search results, please state the alias fields to be returned in the _source property for e.g."
             + " \"_source\":[\"alias.customer\",\"alias.postcode\"]",
-                  required = true)
+            required = true)
         @RequestBody String jsonSearchRequest) {
 
         Instant start = Instant.now();
 
-        rejectBlackListedQuery(jsonSearchRequest);
+        CrossCaseTypeSearchRequest request = elasticsearchQueryHelper.prepareRequest(
```

Comment:
potential regression risk here ⚠️ . We have changed the external API. We introduced:
`buildCaseTypeIds(caseTypeIds)`
on the external API which definitely would require regression testing.